tests/_support/Config/Registrar.php

```diff
@@ -0,0 +1,34 @@
+<?php namespace Tests\Support\Config;
+
+/**
+ * Class Registrar
+ *
+ * Provides a basic registrar class for testing BaseConfig registration functions.
+ */
+
+class Registrar
+{
+
+	public static function Database()
+	{
+		$config = [];
+
+		// Under Github Actions, we can set an ENV var named 'DB'
+		// so that we can test against multiple databases.
+		if ($group = getenv('DB'))
+		{
+			if (is_file(TESTPATH . '_github/Database.php'))
+			{
+				require TESTPATH . '_github/Database.php';
```

Comment:
Sorry, just thinking of this... but let's get rid of the extra file. Now that we have an established Registrar we can work with the settings directly here. It is a better example of how to use a Registrar properly, so could be helpful to developers looking through the code.